Amplitude/Amplitude.cs

```diff
@@ -1,9 +1,11 @@
 ﻿using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Net.Http;
 using System.Text;
 using System.Text.Json;
+using Xamarin.Essentials;
```

Comment:
This is universal library for any runtime. Please, do not use Xamarin or other platform-dependent namespaces.